.github/workflows/appengine-deploy-it.yml

```diff
@@ -10,52 +15,66 @@ jobs:
       with:
         service_account_key: ${{ secrets.APPENGINE_DEPLOY_SA_KEY_JSON }}
         export_default_credentials: true
+    - name: create yaml
+      run: |-
+        cat <<EOF > ${{ github.workspace }}/app.yaml
+        service: "${{ github.job }}-${{ github.run_number }}"
+        runtime: "python37"
+        EOF
     - id: deploy
       name: Deploy to App Engine
       uses: ./appengine-deploy
       with:
         project_id: ${{ secrets.APPENGINE_DEPLOY_PROJECT_ID }}
-        deliverables: ./appengine-deploy/test/app.yaml
+        deliverables: ${{ github.workspace }}/app.yaml
         version: gcloud
     - name: Test Output
-      run: curl '${{ steps.deploy.outputs.url }}'
+      run: curl -sfL '${{ steps.deploy.outputs.url }}'
 
   b64_json:
-    needs: gcloud
     name: with base64 json creds
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: create yaml
+      run: |-
+        cat <<EOF > ${{ github.workspace }}/app.yaml
+        service: "${{ github.job }}-${{ github.run_number }}"
+        runtime: "python37"
+        EOF
     - id: deploy
       name: Deploy to App Engine
       uses: ./appengine-deploy
       with:
         credentials: ${{ secrets.APPENGINE_DEPLOY_SA_KEY_B64 }}
-        deliverables: ./appengine-deploy/test/app.yaml
-        version: b64-json
+        deliverables: ${{ github.workspace }}/app.yaml
+        version: ${{ github.job }}-${{ github.run_number }}-b64-json
         promote: false # Allows for deletion
     - name: Test Output
       run: curl '${{ steps.deploy.outputs.url }}'
     - name: Clean Up
-      run: |
-        gcloud app versions delete b64-json --quiet
+      run: gcloud app versions delete ${{ github.job }}-${{ github.run_number }}-b64-json --quiet
 
   json:
-    needs: [gcloud, b64_json]
     name: with json creds
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: create yaml
+      run: |-
+        cat <<EOF > ${{ github.workspace }}/app.yaml
+        service: "${{ github.job }}-${{ github.run_number }}"
+        runtime: "python37"
+        EOF
     - id: deploy
       name: Deploy to App Engine
       uses: ./appengine-deploy
       with:
         credentials: ${{ secrets.APPENGINE_DEPLOY_SA_KEY_JSON }}
-        deliverables: ./appengine-deploy/test/app.yaml
-        version: json
+        deliverables: ${{ github.workspace }}/app.yaml
+        version: ${{ github.job }}-${{ github.run_number }}-json
```

Comment:
Is this needed if we are adding an unique id for the service?

Reply:
probably not. removed.